README.md

```diff
@@ -1 +1,111 @@
-# python-progress-observer
\ No newline at end of file
+# python-progress-observer
+This library will be used for observing actions of the classes.
+
+**Quick Start**
+
+install the package
+
+    pip install git+ssh://git@bitbucket.org/metglobal/pico.git
+    
+**HOW TO USE IT(BASIC EXAMPLE)**
+
+``` python
+class MyProgressObserver(ProgressObserver):
+    CALLBACKS = ["my_callback"]
+    def my_callback(self, **kwargs):
+        print(f"{message}: [{current_step}/{total_steps}]")
```

Comment:
kwargs üzerinden message bilgisi alırken diğerleri için self kullanılmalı.

Reply:
DONE